core/font.lua

```diff
@@ -42,6 +42,11 @@ SILE.registerCommand("font", function (options, content)
     SILE.settings.set("font.hyphenchar", SU.utf8charfromcodepoint(options.hyphenchar))
   end
 
+  -- We must *actually* load the font here, because by the time we're inside
+  -- SILE.shaper.shapeToken, it's too late to respond appropriately to things
+  -- that the post-load hook might want to do.
+  SILE.font.cache(SILE.font.loadDefaults({}), SILE.shaper.getFace)
```

Comment:
Is this going to mess up the `-d versions` output by outputting information about font files that are not ever actually used? Also won't this error or systems without the default font (Gentium Plus) even in the case of documents or classes which explicitly set a different font early?

Reply:
1) Possibly, and 2) I don't think so, no. Note that this is inside the implementation of the `\font` command, so will only be called if the user (or class) calls that. It won't happen just from setting the default font - in fact, the other way around, it will only happen when a font other than the default is chosen!